src/questions/questions.controller.ts

```diff
@@ -52,7 +108,88 @@ export class QuestionController {
     );
   }
 
-  @Roles(RoleEnum.MANAGER)
+  @Roles(RoleEnum.MANAGER, RoleEnum.ADMIN)
+  @ApiBearerAuth()
+  @UseGuards(JwtAuthGuard, RoleGuard)
+  @Patch('update-question-stack/:stack_id')
```

Comment:
can name this as 'stacks/:stack-id' because FE will understand as in questions, update stacks with id = ?